_data/faq.yml

```diff
@@ -1106,17 +1119,17 @@
 
                 - Start c:geo
                 - Go to c:geo Menu → Settings → System and perform a backup of the database (remember the file location which is shown at the end of the process)
-                - Open a File Explorer, locate the backup file (typically in /sdcard/.cgeo/cgeo.sqlite) and store it at a safe location for later restore on the new device
-                - Optional step: The same directory in which you found the backup file does also contain several folders with cache specific information (pictures, static maps, etc). If this infos are missing on the new device, c:geo will automatically download them again on the next refresh of the cache; so there is no need to copy those. If you however want to avoid that, you can also copy all the subfolders to a safe location for transfer.
-
+                - Open a File Explorer, locate the backup file (typically /cgeo/backup/cgeo.sqlite) and store it at a safe location for later restore on the new device
+                - Optional step: The subfolders of /Android/data/cgeo.geocaching/GeocacheData/ (either located on your internal or your external storage) are containing additional geocache data (e.g. log pictures). If this data is not available on the new device, c:geo will just download it again. If you still want to transfer this data to the new device, copy all subfolders to a safe location.
+                  
                 On the new device:
 
                 - Install the same c:geo version as on the old device
                 - Start c:geo
                 - Store your login credentials and apply your personal settings as needed (those are not part of the backup)
                 - End c:geo
-                - Use a File Explorer to locate the similar folder from which you saved the backup on the old phone (typically sdcard/.cgeo/) and place the backup file (cgeo.sqlite) from the old phone into this folder
-                - Optional step: If you also did a backup of all the subfolders, restore them as well to the same base folder as the backup file
+                - Use a File Explorer to locate the similar folder from which you saved the backup on the old phone (typically /cgeo/backup/) and place the backup file (cgeo.sqlite) from the old phone into this folder. You might have to create the folder manually.
+                - Optional step: If you also did a backup of the additional geocache data, restore it to /Android/data/cgeo.geocaching/files/GeocacheData/ on your internal storage.
```

Comment:
What do you mean?
I explicitly mention the internal storage here, as after a clean installation (on the new device) the internal storage will be the active setting.
I wanted to avoid making it more complicated (e.g. by mentioning the possibility just to place the SD into the new device). 
Furthermore I do not really know what might happen if the user places the old SD into the new device and only then installs c:geo...might this reset the contents of /GeocacheData/ on this SD? Who knows...